source/getting-started/install.rst

```diff
@@ -130,85 +144,80 @@ The installation of Exegol on Linux, macOS and Windows are very similar. It can
+   
+                      sudo python3 -m pip install --requirement "Exegol/requirements.txt"
+   
+               .. tab:: Directly as user
+   
+                   .. code-block:: bash
+   
+                      python3 -m pip install --user --requirement "Exegol/requirements.txt"
+
+    ..  group-tab:: Installing with pipx
+
+        The pre-compiled Exegol’s wrapper can be installed from the PyPI repository. While this is the easiest and most user-friendly technique, it is advised to install from sources, as it allows to switch from release to dev branches easily and it supports the auto-update feature.
+
+        Using ``pipx`` allows you to install Exegol in an **isolated** virtual environment **dedicated** to it.
+
+        First, ``pipx`` must be installed on your host system:
 
         .. code-block:: bash
 
-           python3 -m pip install exegol
-
-        .. warning::
-
-            You may want to disable Windows Defender during the installation, as Exegol will download pre-built remote shells (or temporarily exclude ``C:\Users\<username>\AppData\Local\Temp``).
-
-            You should also add the folder ``C:\Users\<user>\.exegol\exegol-resources`` to the exclution list.
-
-    ..  group-tab:: Installing from sources
+            # install pipx if not already installed, from system package:
+            sudo apt update && sudo apt install pipx
+            # OR from pip
+            python3 -m pip install pipx
 
-        Exegol's wrapper can also be installed from sources (with Git). The wrapper then knows how to self-update, and switching from release and development branches is possible and very easy.
+        Exegol's wrapper can be installed with ``pipx`` either from **sources** or pre-compiled from **PyPI**:
 
         .. code-block:: bash
 
-           git clone "https://github.com/ThePorgs/Exegol"
-
-        .. tip::
-
-            If you want a **light** clone of Exegol (and **never** use the **dev** branch), you can use the following command:
-
-            .. code-block:: bash
-
-                git clone --shallow-since="2023/05/08" "https://github.com/ThePorgs/Exegol"
-
-        If you have access to docker directly as a user, you can install the requirements only for your current user
-        otherwise the requirements must be installed as root to run Exegol with sudo.
-
-        .. tabs::
-
-            .. tab:: With sudo
-
-                .. code-block:: bash
-
-                   sudo python3 -m pip install --requirement "Exegol/requirements.txt"
-
-            .. tab:: Directly as user
-
-                .. code-block:: bash
+            # You can now install Exegol package from PyPI
+            pipx install exegol
 
-                   python3 -m pip install --user --requirement "Exegol/requirements.txt"
+            # Or from sources directly
+            pipx install git+https://github.com/ThePorgs/Exegol
 
-    ..  tab:: Installing with pipx
+    ..  group-tab:: Installing with pip
```

Comment:
Maybe it would be relevant to add a warning section for those that will encounter the error where Ubuntu says the package should installed in its own venv ?

Reply:
By using pipx as recommended we shoudn't have this problem